diff --git a/lib/client.js b/lib/client.js
index 6ff210e..7bd36ed 100644
--- a/lib/client.js
+++ b/lib/client.js
@@ -200 +200 @@ window.__ModuleLoader__.load({
-						children: renderSlot("sidebar.workspaces", {
+						children: [renderSlot("sidebar.workspaces", {
@@ -205 +205 @@ window.__ModuleLoader__.load({
-						})
+						}), renderSlot("sidebar.footer.action", { wide, expandSidebar: () => { if (collapsed) toggleSidebar(); } })]
@@ -211 +211 @@ window.__ModuleLoader__.load({
-							children: renderSlot("sidebar.footer.action", { wide })
+							children: null
